src/js/rx/containers/Main.jsx

```diff
@@ -12,15 +12,13 @@ class Main extends React.Component {
   render() {
     let alertBox;
 
-    if (this.props.alert.visible) {
-      alertBox = (
-        <AlertBox
-            content={this.props.alert.content}
-            isVisible={this.props.alert.visible}
-            onCloseAlert={this.props.closeAlert}
-            status={this.props.alert.status}/>
-      );
-    }
+    alertBox = (
+      <AlertBox
```

Comment:
It seems like a useful component to have.  I'd rather fix it in `./common` than reinvent the wheel... I think I'll do that